File: perl/g10.array.pl
#!/usr/bin/perl -w
# declare and initilize list on one line
my @nicknames=qw(bob ed steve ralph joe deb kate);
# output the results of list
print "The names are: \n";
# use collection loop to get items in list
foreach my $name (@nicknames) {
 print " $name\n";
}

File: compiled_lang/go/src/g10.array.go
package main

import "fmt"

func main() {
	nicknames := []string{"bob", "ed", "steve", "ralph", "joe", "deb", "kate"}

	fmt.Println("The names are: ")
	for _, name := range nicknames {
		fmt.Println("  " + name)
	}
}
